### PR TITLE
Generate top_level_request_id as an integer rather than uuid

### PR DESCRIPTION
**Pull request by abhinab-yb**

### CODE REVIEW

File: src/yb/util/oid_generator.h [hunk 1]
```diff
@@ -38,4 +38,70 @@ namespace yb {
...
+
+  static constexpr uint64_t min() noexcept { return 0; }
+
+  static constexpr uint64_t max() noexcept { return std::numeric_limits<uint64_t>::max(); }
+
+private:
+  std::array<uint64_t, 2> state_{};
+};
+
+class OtelRandom
```
Comment (vrajat) [inline src/yb/util/oid_generator.h:87]: I suggest you remove otel from the name. In a comment link to the source code where you picked it up from.

Comment (abhinab-yb) [reply src/yb/util/oid_generator.h:87]: There is already a Random class, that's why I changed the name

File: src/postgres/src/backend/catalog/yb_system_views.sql [hunk 2]
```diff
@@ -1000,9 +1000,9 @@ CREATE VIEW pg_replication_origin_status AS
 
 REVOKE ALL ON pg_replication_origin_status FROM public;
 
-CREATE VIEW tserver_stat_activity AS
-    SELECT *
-    FROM tserver_stat_get_activity();
+-- CREATE VIEW tserver_stat_activity AS
```
Comment (vrajat) [inline src/postgres/src/backend/catalog/yb_system_views.sql:1003]: Why is this commented out?

File: src/postgres/src/include/pg_yb_utils.h [hunk 3]
```diff
@@ -879,6 +879,23 @@ void YbSetIsBatchedExecution(bool value);
 #endif
 
 void ProcSetNodeUUID(const char *);
-void ProcSetTopRequestId(const char *);
+void ProcSetTopRequestId(const uint64_t *);
+
+static void
```
Comment (vrajat) [inline src/postgres/src/include/pg_yb_utils.h:884]: Is this fn - also copied from otel?

File: src/yb/util/oid_generator.h (hunk 1 above)
Comment (vrajat) [reply src/yb/util/oid_generator.h]: Can you call it AUHRandom?

File: src/postgres/src/include/pg_yb_utils.h (hunk 3 above)
Comment (abhinab-yb) [reply src/postgres/src/include/pg_yb_utils.h:884]: no, it's your fn to set it in PGPROC

File: src/postgres/src/backend/catalog/yb_system_views.sql (hunk 2 above)
Comment (abhinab-yb) [reply src/postgres/src/backend/catalog/yb_system_views.sql:1003]: forgot to uncomment, fixing

File: src/yb/util/oid_generator.h (hunk 1 above)
Comment (abhinab-yb) [reply src/yb/util/oid_generator.h]: yeah

**Pull request merged**